using-mobile-extensions/adobe-analytics/analytics-api-reference.md

```diff
@@ -1009,7 +1150,36 @@ Analytics.sendQueuedHits();
 ```
 {% endtab %}
 
-{% tab title="iOS" %}
+{% tab title="iOS (AEP 3.x)" %}
+
+### sendQueuedHits
+
+Regardless of how many hits are currently queued, this method forces the library to send all hits in the offline queue.
```

Comment:
This method forces the library to send all hits in the offline queue, regardless of how many hits are currently queued.